lib/saluki-event/src/service_check/status.rs

```diff
@@ -0,0 +1,15 @@
+/// ServiceCheckStatus is the status type for service checks
+#[derive(Clone, Debug)]
+pub enum ServiceCheckStatus {
+    /// Ok is the "ok" ServiceCheck status
```

Comment:
Similar comment to the above.

For example, you might say "The service is operating normally." for the `Ok` variant, and then say "The service is in a warning state." for `Warning` state, and so on.

Basically, if we're going to restate what is likely already obvious, we should just do it in a way we'd naturally say it.

Reply:
gotcha, was just copying from the [go code ](https://github.com/DataDog/datadog-go/blob/1139efe797e4760daf3d30a60e3d30afc1687df8/statsd/service_check.go#L12) 😅